reframe_config.py

```diff
@@ -13,16 +13,20 @@
                     'descr': 'CSD3 Login nodes',
                     'scheduler': 'local',
                     'launcher': 'local',
-                    'environs': ['gnu'],
+                    'environs': ['gnu','intel'],
```

Comment:
I don't think we need to add ReFrame programming environments: we aren't planning to use ReFrame's own build system at any point, Spack will deal with all of that, and that's independent from these programming environments.